package.json

```diff
@@ -175,6 +175,7 @@
     "rtlcss": "^2.6.0",
     "sanitize-filename": "^1.6.3",
     "serialize-javascript": "^5.0.1",
+    "sharp": "^0.29.2",
```

Comment:
Can you please install sharp as a devDependency instead, please?

Reply:
Ooh yea

Reply:
Done